addon/components/power-select.js

```diff
@@ -570,6 +573,13 @@ export default Component.extend({
     }
   },
 
+  _cancelActiveSearch() {
+    let publicAPI = this.get('publicAPI');
+    if (publicAPI._activeSearch && typeof publicAPI._activeSearch.cancel === 'function') {
+      publicAPI._activeSearch.cancel();
+    }
+  },
+
```

Comment:
FYI, Ember has a `tryInvoke` that does just this without the boilerplate.

```js
let activeSearch = this.get('publicAPI._activeSearch');
Ember.tryInvoke(activeSearch, 'cancel');
```

http://emberjs.com/api/#method_tryInvoke

Also I just checked and `tryInvoke` is 'null' / 'undefined' safe.